packages/backend/index.ts

```diff
@@ -63,14 +62,26 @@ app.get('/myApi/auth/callback', (req: Request, res: Response) => {
 });
 
 app.get('/myApi/auth/token', (req, res) => {
-  console.log("request object in /myApi/auth/token", req);
   res.json(
      {
         access_token: access_token
      })
 })
-app.get('/myApi/search', (req, res) => {
+app.get('/myApi/search', (req: Request, res: Response) => {
   console.log('hit search endpoint');
+  const auth_query_parameters = new URLSearchParams({
+    type: 'album,track',
+    include_external: 'audio',
+    limit: '5',
+    q: req.query.q as string
+  })
+  axios.get('https://api.spotify.com/v1/search?' + auth_query_parameters.toString(), {
+    headers: {
+      'Authorization': "Bearer " + access_token
```

Comment:
This is super minor, but you can use a template literal here as such:

"Authorization": `Bearer ${access_token}`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals